en/docs/learn/multi-factor-authentication-for-wso2-is-management-console.md

```diff
@@ -10,6 +10,16 @@
     2.  **Assertion Consumer URLs**: `https://localhost:9443/acs`
     3.  **Enable Response Signing**: `true`
 
+        !!! Tip 
+           
+            When configuring single-sign-on for Management Console, "Issuer" value of the SAML request is default to **carbonServer** as mentioned above.
```

Comment:
When configuring single-sign-on for the Management Console, the "Issuer" value of the SAML request is defaulted to **carbonServer** as mentioned above.